src/icon-button.jsx

```diff
@@ -28,6 +32,26 @@ const IconButton = React.createClass({
     };
   },
 
+  statics: {
+    getRelevantContextKeys(muiTheme) {
```

Comment:
Cool, I've merged the new theming documentation in #1716. So make sure to rebase if you need to!